### PR TITLE
Custom emails for specific project

### PR DESCRIPTION
**Pull request by JMurphyWeb**
ready

- [x] Update project id in email templates before deploying

Refactor email templates.

#151

### CODE REVIEW

**Comment by jonscottfilm** [comment]
need to swap buttons at bottom of 'Make copy' page, match the one's at the top:

Make these:

image


Like these:

image

**Comment by jonscottfilm** [comment]
Non-Pathe emails look good, except for:

Space in front of greeting on Create account and New offer emails:

' Hi!'

**Comment by jonscottfilm** [comment]
Notes from testing:

1. In Revised offer email. Amend first line to be: 
Your offer, subject to contract, to join 'Test 10' has been updated

2. CERTIFICATE OF ENGAGEMENT (or COE) to also be in same colour red, in all templates.

3. Grey header in 3 emails (new, revised, accepted): **SUBJECT TO CONTRACT**

File: lib/ev2_web.ex [hunk 1]
```diff
@@ -34,7 +34,7 @@ defmodule Ev2Web do
 
   def view do
     quote do
-      use Phoenix.View, root: "lib/ev2_web/templates",
+      use Phoenix.View, root: "lib/ev2_web/templates", pattern: "**/**",
```
Comment (jackcarlisle) [inline lib/ev2_web.ex]: What's this doing?

Comment (JMurphyWeb) [reply lib/ev2_web.ex]: This allows us to use folders in our templates.

Take a look at the templates/email/ folder

**Pull request merged**